HW2.js

```diff
@@ -0,0 +1,80 @@
+//#1
+
+console.log(durationBetweenDates("31 Jan 2022", "03 Feb 2021", "days"));
+
+function durationBetweenDates(
+    startDate = new Date(),
+    endDate = new Date(new Date().getDate() + 1), 
+    units = "days") {
+
```

Comment:
```suggestion
function durationBetweenDates(
    startDate = new Date(),
    endDate = new Date(new Date().getDate() + 1), 
    units = "days"
    ) {

```

Стилістично краще в такому випадку перенести